test/radix/quic_ops.c

```diff
@@ -298,6 +335,18 @@ DEF_FUNC(hf_accept_conn)
         goto err;
     }
 
+    if (check_pending_match) {
```

Comment:
Should we test that the callback has already been called prior to `SSL_accept_connection` being called?

Reply:
Unless something goes wrong during the test, and the connection fails, I don't think that will ever happen, but it can't hurt

Reply:
I think you misunderstood what I meant. My point was that we expect that the user SSL will be proactively created early, and that SSL_accept_connection() will just return the thing that we already created. Therefore I might expect that immediately before the call to `SSL_accept_connection` above, `pending_cb_called` should already be set?

Reply:
yes, it absolutely should.  The callback could be made by any previous SSL call that triggers a call to the quic reactor.  Hence the flags to indicate that the callback was triggered for the given script.

Reply:
Hence my question - can we test that, i.e. add a check immediately before `SSL_accept_connection` that `pending_cb_called` is true. This would give me some confidence that things are working the way I expect.

Reply:
We....could, but theres no guarantee that it will have happened before the call to SSL_accept_connection is made.  Its entirely possible that, due to timing, we call SSL_accept_connection->ossl_quic_accept_connection->qctx_maybe_autotick().

The call to tick the quic reactor may just as likely occur in a prior SSL call that ticks the reactor, but possibly not, and both are valid.  I'd be concerned that checking for the callback being called prior to the accept call would lead to random false positive failures.

Reply:
We would have to use SSL_poll() to check whether we have an incoming connection and only after that call SSL_accept_connection().

Reply:
or some other call that ticks the quic reactor, yes, assuming we wanted to do that, but I'm not sure thats worthwhile.  All we want to do is make sure that the callback to new_pending_ssl is made, I don't think it matters that much if it happens as an artifact of SSL_accept_connection, or some prior call like SSL_poll, as both triggers are perfectly valid